src/calculateRentalCost.js

```diff
@@ -28,7 +28,9 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  let result = days * 40;
+
+  return (days >= 7) ? (result -= 50) : (days >= 3) ? (result -= 20) : result;
```

Comment:
Use constants that explain values, Checklist(2). Other than that all great !

Reply:
Thank you, now I know how it's supposed to be